src/nrg_gas_utilities/src/nrg_gas_utilities/nrg_gas.cpp

```diff
@@ -51,7 +52,8 @@ Point NRGGas::calculateSourceTransform(const PointStamped& source, const Vector3
     // TODO- why do we need 2*M_PI?
     q_wind_azimuth.setRPY( 0, 
                            0, 
-                           2*M_PI-atan2(wind.vector.y, wind.vector.x));       // Wind azimuth in anemometer frame
+                           //2*M_PI-atan2(wind.vector.y, wind.vector.x)
+                           wind.azimuth);       // Wind azimuth in anemometer frame
```

Comment:
Do not comment out the line, delete the line. You will need to confirm that this azimuth is the same as the one I calculated. How will you do that?